src/core/load-fragment-gperf.gperf.in

```diff
@@ -2,267 +2,268 @@
 
 {%- macro EXEC_CONTEXT_CONFIG_ITEMS(type) -%}
 {# Define the context options only once #}
-{{type}}.WorkingDirectory,                 config_parse_working_directory,              0,                                  offsetof({{type}}, exec_context)
```

Comment:
BTW, please split out such reindentation into another commit.
Otherwise, the commit becomes huge and is filled with mostly uninteresting change...